### PR TITLE
Feat/ship corpse

### PR DESCRIPTION
**Pull request by mRmetr0**
Please check if you can kill a ship and collect the wreck (with points)

### CODE REVIEW

File: BattleShips_Host/Assets/Scripts/Shipwreck.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Shipwreck : MonoBehaviour
+{
+    private int treasure;
+
+    public int Treasure => treasure;
```
Comment (TechnicJelle) [inline BattleShips_Host/Assets/Scripts/Shipwreck.cs:9]: You can swap this whole bit out for `public int Treasure { get; private set; }`

**Review by TechnicJelle — CHANGES_REQUESTED** [review]
Looking good already, but there are a couple things that still caught my eye:

File: BattleShips_Host/Assets/Scripts/Boat.cs [hunk 2]
```diff
@@ -151,6 +153,14 @@ private void OnTriggerEnter(Collider other)
 			CollidingIsland = other.gameObject;
 			WebsocketServer.Instance.Send(_id, MessageFactory.CreateDockingAvailableUpdate(true));
 		}
+
+		if (other.gameObject.CompareTag("Shipwreck"))
+		{
+			int currentPoints = MatchManager.Instance.GetPlayerPoints(_id);
+			int shipPoitns = other.GetComponent<Shipwreck>().Treasure;
```
[the remark this answers is not in the document]
Comment (mRmetr0) [reply BattleShips_Host/Assets/Scripts/Boat.cs]: Should be fixed now

**Review by TechnicJelle — APPROVED** [review]
lgtm now!

**Pull request merged**